excel/src/robocorp/excel/__init__.py

```diff
@@ -2,7 +2,7 @@
 Main module for doing Excel automation.
 
 This library can be made available by pinning
-![`robocorp-excel`](https://img.shields.io/pypi/v/robocorp-excel?label=robocorp-excel)
+[![`robocorp-excel`](https://img.shields.io/pypi/v/robocorp-excel?label=robocorp-excel)](https://pypi.org/project/robocorp-excel/)
```

Comment:
We want these clickable anywhere in the handwritten or generated docs. (staying consistent with the rest of the pins)